basyx.submodelrepository-http/src/test/java/org/eclipse/digitaltwin/basyx/submodelrepository/http/TestSubmodelRepositoryHTTP.java

```diff
@@ -75,21 +76,21 @@ public void shutdownAASRepo() {
 	@Test
 	public void getAllSubmodelsPreconfigured() throws IOException, ParseException {
 		String submodelsJSON = getAllSubmodelsJSON();
-		String expectedSubmodelsJSON = getAllSubmodelJSONExpected();
+		String expectedSubmodelsJSON = getAllSubmodelJSON();
 		assertSameJSONContent(expectedSubmodelsJSON, submodelsJSON);
 	}
 
 	@Test
 	public void getSpecificSubmodel() throws ParseException, IOException {
-		String submodelJSON = getSpecificSubmodelJSON(DummySubmodelFactory.createTechnicalDataSubmodel().getId());
-		String expectedSubmodelJSON = getSpecificSubmodelJSONExpected();
+		String submodelJSON = requestSpecificSubmodelJSON(DummySubmodelFactory.createTechnicalDataSubmodel().getId());
+		String expectedSubmodelJSON = getSingleSubmodelJSON();
 
 		assertSameJSONContent(expectedSubmodelJSON, submodelJSON);
 	}
 
 	@Test
 	public void getSpecificSubmodelNonExisting() throws IOException {
-		CloseableHttpResponse response = getSubmodel("nonExisting");
+		CloseableHttpResponse response = requestSubmodel("nonExisting");
```

Comment:
Not sure if necessary but shouldn't a `CloseableHttpResponse `(and `CloseableHttpClient`) finally be closed? (either in finally block or in try-with-resources)